### ISSUE TITLE
Not able to assign IP address after built and boot loader

### ISSUE THREAD

**Issue by Tonetete** [fbelavenuto/arpl#162]
Hello there,

Already posted on xpenogloy forum:

First thanks for the automated solution. I'm having the following issue, when I choose (DS3622xs+) and build it and when I boot the loader, I lose the IP assignation immediately and I'm unable to init synology assistant to start the installation and when I reboot and loader it's booted I get an ERROR on ip assignation. Do I miss some specific network driver or something like this? I'm using an HP Proliant Microserver Gen10 (AMD opteron X3126)

The NIC brand is an Embedded Broadcom BCM5720 Dual Port Gbe LOM.

Tried with alpha 5, 6 and 7 versions, same results.

**Comment by Tonetete** [comment]
Hello again,

I already tried with alpha 7 as you can see in the picture.

**Comment by fbelavenuto** [comment]
Yes, but this alpha7 is bugged! I've fixed the bug and recreated alpha7 release!

**Comment by Tonetete** [comment]
Hello again,

So I downloaded the new img release, burnt it and booted. Same steps: choose model, version, generate random serial number, check config, build the loader and then boot loader and again, I lose IP assignation...

Tried also to assign IP manually in DHCP of my router to NAS mac and also not connection.

I'm running out of ideas...

**Comment by Tonetete** [comment]
One thing to mention is immediately the bootloader it's loaded in the very first time, the IP assignation returns error  but the router assign the IP anyway so I can enter the menu without running `menu.sh` however after build the loader and reboot, I'm not longer able to access it unless I access by enter `menu.sh` and the NIC is detected again.

**Comment by sm622** [comment]
Same issue with me I am using v0.4-alpha7

**Comment by fbelavenuto** [comment]
Ok, I'll investigate a new driver for this network card (BCM5720)

**Comment by masyanich** [comment]
same problem here, but lan nic is Realtek Semiconductor Co., Ltd. RTL8111/8168/8411 P                                                     CI Express Gigabit Ethernet Controller (rev 15), can you please add drivers for this card family?

**Comment by OpenNEONE** [comment]
Hello, I have encountered the same problem, for which I have rebuilt almost all versions and models. My network card model is r8250, there is really no way

**Comment by edhnt455** [comment]
Hello, I have encountered the same problem too. My equipment is the j3455 of Asrock, the network card model is Realtek RTL8111GR. I hope you can solve this problem. Thank you very much！

**Comment by OpenNEONE** [comment]
> Hello, I have encountered the same problem too. My equipment is the j3455 of Asrock, the network card model is Realtek RTL8111GR. I hope you can solve this problem. Thank you very much！

You can try to cut off all external networks when using the pat file DSM, and unplug the boot U disk, then wait quietly for 10 minutes before restarting manually. Because DSM7+ may modify the files of the boot U disk or upgrade the version during installation, the driver may be lost.

**Comment by fbelavenuto** [comment]
> Hello, I have encountered the same problem, for which I have rebuilt almost all versions and models. My network card model is rtl8125, there is really no way

I have a USB adapter with the RTL8153 chip and I couldn't get its driver to work! I tested the latest versions I found for the DSM kernel (4.4.180) but still no success.

**Comment by edhnt455** [comment]
> > Hello, I have encountered the same problem too. My equipment is the j3455 of Asrock, the network card model is Realtek RTL8111GR. I hope you can solve this problem. Thank you very much！
> 
> You can try to cut off all external networks when using the pat file DSM, and unplug the boot U disk, then wait quietly for 10 minutes before restarting manually. Because DSM7+ may modify the files of the boot U disk or upgrade the version during installation, the driver may be lost.

Unfortunately, it didn't work.

**Comment by fbelavenuto** [comment]
Please update the modules and recreate the loader. I put new drivers
